optuna/storages/rdb/storage.py

```diff
@@ -629,3 +682,40 @@ def __del__(self):
 
         if hasattr(self, 'scoped_session'):
             self.remove_session()
+
+
+class _FinishedTrialsCache(object):
+    def __init__(self, enabled):
+        # type: (bool) -> None
+
+        self._finished_trials = {}  # type: Dict[int, structs.FrozenTrial]
+        self._enabled = enabled
+        self._lock = threading.Lock()
+
+    def _is_empty(self):
```

Comment:
I have a question about naming convention.
When we create a private class, do we also make the methods private like `_FinishedTrialsCache`?
In this case, I think their names can be public like `is_empty` since the methods of `_FinishedTrialsCache` are called by RDBStorage.

Reply:
Thank you for your suggestion.
It sounds reasonable, so I adopted the suggested naming convention (https://github.com/pfnet/optuna/pull/349/commits/8d33c54ac27b4c0a58f8146445066bac99177024).